specs/Appearance.md

```diff
@@ -0,0 +1,93 @@
+Theme API
+===
+
+# Background
+This API's main use is to set the overall appearance/theme for WebView2. The options are similar to Edge: match the system default theme, change to light theme, or change to dark theme. 
+This API has 2 main changes relevant to the end users. First, it sets appearance for WebView2 UI like dialogs, prompts, context menu, etc. And second, this API sets the ['prefers-color-scheme'](https://developer.mozilla.org/en-US/docs/Web/CSS/@media/prefers-color-scheme) [media feature](https://developer.mozilla.org/en-US/docs/Web/CSS/Media_Queries/Using_media_queries#media_features). Websites typically media query for this setting in order to set CSS themes for light/dark. 
+
+Please note this API will only set the overall appearance, but not theme as defined in the Edge browser.
+For reference, in the screenshot below, this API is meant to expose the Overall Appearance section as a WebView2 API. 
+
+![Alt text](./media/EdgeSettingsAppearance.png "a title")
+# Examples
+
+### C++
+
+```cpp
+
+void ViewComponent::SetTheme(COREWEBVIEW2_THEME_KIND value)
+{
+    
+    wil::com_ptr<ICoreWebView2Controller4> webViewController4;
+
+    m_appWindow->GetWebViewController()->QueryInterface(IID_PPV_ARGS(&webViewController4));
+    if (webViewController4)
+    {
+        CHECK_FAILURE(webViewController4->put_Theme(value));
+    }
+}
+    
+```
+### C#
+
+```c#
+
+private WebView2 m_webview;
+
+void SetTheme(COREWEBVIEW2_THEME_KIND value)
```

Comment:
The enum won't look like this in WinRT/C#. After you update the WinRT MIDL3 below with the enum definition you can see its more like CoreWebView2ThemeKind for the type
```suggestion
void SetTheme(COREWEBVIEW2_THEME_KIND value)
```